dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_number_input.tsx

```diff
@@ -27,6 +27,7 @@ export const InputFieldItem: React.FC<InputFieldProps> = ({
       fullWidth
       placeholder="auto"
       value={numValue}
+      min={1}
```

Comment:
it will be applied to all charts wherever this component is being used. Kindly verify once.

Reply:
Yes, I have verified it everywhere